tests/test_protocol.py

```diff
@@ -2,6 +2,8 @@
     Point, Range, Request, Notification
 )
 import unittest
+from LSP.plugin.core.transports import _encode, _decode
```

Comment:
that import section could use some cleanup :)

Reply:
I'm playing around with enabling `black` and `isort` for this stuff